python/ray/workflow/step_executor.py

```diff
@@ -485,52 +236,41 @@ def _workflow_step_executor_remote(
     """The remote version of '_workflow_step_executor'."""
     with workflow_context.workflow_logging_context(job_id):
         return _workflow_step_executor(
-            func, context, job_id, step_id, baked_inputs, runtime_options
+            func, context, step_id, baked_inputs, runtime_options
         )
 
 
 def _workflow_wait_executor(
```

Comment:
should we revert this and move the change to the next PR given that we don't support wait for now?